app/com/yetu/app/com/yetu/youtube/views/index.scala.html

```diff
@@ -0,0 +1,49 @@
+@(devToken: String)(authUrl: String)(pollingInterval: Int)
```

Comment:
As you have changed the path to this file, please make sure, that gulp is also configured in the right way. I do not see any gulp files changed. Normally in the config file for the gulp path a change should be done for the views paths.